homeassistant/components/aussie_broadband/__init__.py

```diff
@@ -30,6 +42,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         entry.data[CONF_PASSWORD],
         async_get_clientsession(hass),
     )
+    # override the packaged method because pydantic 2.x isn't supported yet
```

Comment:
Please update the comment and mention what library version we're currently using so we know when the override is valid.